src/components/VisualizationTypeSelector/styles/VisualizationTypeSelector.module.css

```diff
@@ -3,17 +3,29 @@
     margin-left: auto;
 }
 
+.arrowIcon.listIsOpen {
+    transform: rotate(180deg);
+}
```

Comment:
I'm now rotating the arrow-icon when the list is open, as we do on select elements to give more visual feedback to the user and make the UI more consistent.